AxonDeepSeg/data_management/dataset_building.py

```diff
@@ -53,15 +57,15 @@ def raw_img_to_patches(path_raw_data, path_patched_data, thresh_indices = [0, 0.
             to_extract = [img, mask]
             patches = extract_patch(to_extract, patch_size)
             # The patch extraction is done, now we put the new patches in the corresponding folders
-            
+
             # We create it if it does not exist
-            path_patched_folder = os.path.join(path_patched_data,img_folder)
-            if not os.path.exists(path_patched_folder):
-                os.makedirs(path_patched_folder)
-            
+            path_patched_folder = path_patched_data / img_folder
+            if not path_patched_folder.exists():
+                path_patched_folder.mkdir(parents=True)
+
             for j, patch in enumerate(patches):
-                imsave(os.path.join(path_patched_folder,'image_%s.png'%j), patch[0],'png')
-                imsave(os.path.join(path_patched_folder,'mask_%s.png'%j), patch[1],'png')
+                imsave(path_patched_folder.joinpath('image_%s.png'%j), patch[0],'png')
```

Comment:
Why use this syntax instead of something like `path_patched_folder / 'image_%s.png'%j`?

Reply:
I see, that syntax doesn't work. Then how about:

```python

tmp_image = 'image_' + str(j) + '.png'

imsave(path_patched_folder / tmp_image, patch[0],'png')
```

That way we're consistent with how paths are joined. Same goes for the line below (and maybe other files, I haven't covered them yet)